src/bidiMapper/domains/events/EventManager.ts

```diff
@@ -265,7 +268,7 @@ export class EventManager implements IEventManager {
   #markEventSent(
     eventWrapper: EventWrapper,
     channel: string | null,
-    eventName: string
+    eventName: Session.SubscribeParametersEvent
```

Comment:
And I believe here it should be the specific event as well
```suggestion
    eventName: Message.EventNames
```